addons/account_audit_trail/models/mail_message.py

```diff
@@ -197,7 +197,7 @@ def _except_audit_log(self):
     def write(self, vals):
         if (
             vals.keys() & {'res_id', 'res_model', 'message_type', 'subtype_id'}
-            or ('subject' in vals and any(self.mapped('subject')))
+            or ('subject' in vals and any(' '.join(s.subject.splitlines()) != vals['subject'] for s in self if s.subject))
```

Comment:
(There may be better ways:)
```suggestion
            or ('subject' in vals and any(' '.join(s.subject.split()) != ' '.join(vals['subject'].split()) for s in self if s.subject))
```
But maybe normalize the `vals['subject']` outside the loop (not sure it matters much)